src/main/java/md/tekwillacademy/gitworkflow/GitWorkFlow.java

```diff
@@ -0,0 +1,40 @@
+package md.tekwillacademy.gitworkflow;
+
+public class GitWorkFlow {
+    public static void main(String[] args) {
+        String step1 = "Perform git stash - to store the uncommitted changes";
+        String step2 = "Perform git checkout master - to move to the local master branch";
+        String step3 = "Perform git fetch - sync with origin/master";
+        String step4 = "Perform git pull - to retrieve changes from a remote repository";
+        String step5 = "Perform git checkout -b new_branch_name - to create and instantly move to a new branch";
+        String step6 = "Add code changes that related to the task";
+        String step7 = "Perform git commit - to save the changes on local memory";
+        String step8 = "Perform git push - to upload the changes on the remote repository(cloud)";
```

Comment:
Thank you Alex, please add the following steps:

String step_12_1 = "If changes are required then I have to modify my local branch and perform a git commit amend";
String step_12_2 = "After git commit amend, we have to do a git push -f ";

System.out.println(step_12_1);
System.out.println(step_12_2);